packages/app-admin/src/plugins/Menu/Navigation/index.tsx

```diff
@@ -55,6 +53,18 @@ const Navigation = () => {
             );
         });
 
+    const footerMenus = [];
+    const footerMenuPlugins = getPlugins<AdminDrawerFooterMenuPlugin>("admin-drawer-footer-menu");
+
+    footerMenuPlugins &&
+        sortBy(footerMenuPlugins, [p => p.order || 50, p => p.name]).forEach(plugin => {
```

Comment:
@BhuwanChandra I don't think we need `sortBy` here as we don't define `order` property on the plugin itself.